scripts/coverage.sh

```diff
@@ -69,7 +72,6 @@ grcov_common_args=(
   --ignore upload-perf\*
   --ignore bench-streamer\*
   --ignore local-cluster\*
-  --ignore-not-existing
```

Comment:
let's see what happens. I'd like to avoid these squash-error flags if possible.

Reply:
@yihau seems this isn't needed for now from the fact the coverage step passed green? If you think this is still needed, could you share the reason?

Reply:
no. I think it is good to remove it if we can pass the test without it!